modules/assim.batch/R/pda.emulator.R

```diff
@@ -55,10 +64,10 @@ pda.emulator <- function(settings, params.id = NULL, param.names = NULL, prior.i
     con <- NULL
   }
 
-  bety <- PEcAn.visualization::betyConnect("~/pecan/web/config.php")
+  bety <- betyConnect("~/pecan/web/config.php")
```

Comment:
Definitely!  I suspect @istfer was just copying the design pattern from elsewhere, but this could easily fail if someone installs pecan somewhere other than HOME